src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -614,6 +614,8 @@ public static DynamoModel Start(IStartConfiguration configuration)
         /// <param name="config">Start configuration</param>
         protected DynamoModel(IStartConfiguration config)
         {
+            DynamoModel.IsCrashing = false;
+
```

Comment:
we were never resetting the DynamoModel.IsCrashing flag.

Reply:
That's because it will default to false each time Dynamo restarts, won't it?

Reply:
It is a static. It was not being reset anywhere

Reply:
Aren't static properties defaulted to their default values or is that true only for non-static properties and fields?

Reply:
This is assuming Dynamo will crash after iscrashing is set to true, and Dynamo needs to restart to reset this value. In any case, minor point.

Reply:
In some cases Dynamo4Revit can crash but leave REvit running.
Dynamo will close and generate a CER report.
Then Dynamo Could be opened up again from the same session of Revit